perf.groovy

```diff
@@ -37,3 +55,61 @@ def projectFolder = Utilities.getFolderName(project) + '/' + Utilities.getFolder
         }
     }
 }
+
+// Create the Linux/OSX/CentOS coreclr test leg for debug and release and each scenario
+[true, false].each { isPR ->
+    ['Ubuntu'].each { os ->
+        def osGroup = getOSGroup(os)
+        def architecture = 'x64'
+        def configuration = 'Release'
+        def newJob = job(Utilities.getFullJobName(project, "perf_${osGroup}", isPR)) {
+            steps {
+                // Set up the copies
+                // Coreclr build containing the tests and mscorlib
+                shell('curl http://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/master/job/release_windows_nt/lastSuccessfulBuild/artifact/bin/tests/tests.zip -o tests.zip')
```

Comment:
What you are mentioning are the exact reasons why I would prefer this to be hoisted. Having to go manually add and configure a job to test it is not scalable. Doing that is a manual process that would have to be done every time you update these build commands potentially corrupting the job history. 

Almost everything we have done with dotnet-ci follows the general rule: get it working locally, then add it to automation. I understand as well that you are saying changing the xunit perf script was difficult because it lead to many changes to the checked in file. To me this sounds like the xunit perf script itself should be more easily testable locally. There is no reason you should be unable to provide the same validation locally that is done in the lab. If this assumption is made then the script's validation and debugging would not require the lab's job.